src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir.ttc);
+}
+
+form {
+  margin-top: 20px;
+}
+
+.bar {
+  width: 100%;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.barLarge {
+  height: 70px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  padding: 0 62px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.barSmall {
+  height: 42px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  font-style: normal;
+  padding: 0 33px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
+
+.lupaLarge {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  padding: 25px 17px 26px 26px;
+}
+
+.lupaSmall {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  padding: 15px 9px 16px 13px;
+}
```

Comment:
Omg, you duplicate a lot of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item